.dotnet/example_code/DynamoDB/TryDax/05-Scan-Test.cs

```diff
@@ -25,28 +25,19 @@
 using Amazon.Runtime;
```

Comment:
Add copyright notice.

Reply:
Revised as per discussion to match approved samples in TryDax.zip.